docs/compute/numa.md

```diff
@@ -13,16 +13,17 @@ The following NUMA mapping strategies can be used:
 
 - [**GuestMappingPassthrough**](#guestmappingpassthrough)
 
+**FEATURE STATE:** KubeVirt v1.4 GA
+
+The `NUMA` feature gate is now deprecated and as such enabled by default. The following [preconditions](#preconditions) still apply however.
+
```

Comment:
Should this be in place of line 3?

Reply:
I wanted to retain the history of the feature thus the additional block.

We've not introduced any additional strategies yet but the existing functionality is mature enough that it doesn't require the feature gate anymore.